source/user/mcscf.rst

```diff
@@ -1,57 +1,299 @@
+  mycas = mcscf.CASSCF(myhf, 6, 8)
+  mycas.frozen = [0,1,26,27]
+  mycas.kernel()
+
+See :source:`examples/mcscf/19-frozen_core.py` for a complete example.
+
+
+State-Averaged Calculations
+---------------------------
+
+When dealing with states that are close in energy, it can be helpful to perform state average calculations where the multireference orbitals are optimized for multiple states.
+The ``state_average_`` function (note the hanging underscore) is a member function of ``CASCI``/``CASSCF`` objects and takes the weights of the states as input.
+The weights can be any normalized and non-negative array of values.
+
+.. code-block:: python
+
+  n_states = 5
+  weights = np.ones(n_states)/n_states
+  mycas = mcscf.CASSCF(mf, 4, 4).state_average_(weights)
+
+See :source:`examples/mcscf/15-state_average.py` for a complete example.
+
+
+By default, only a single spin and/or point group symmetry is targeted, but it is possible to target a mixture of both:
+
+.. code-block:: python
+
+  weights = [.5, .5]
+  solver1 = fci.direct_spin1_symm.FCI(mol)
+  solver1.wfnsym= 'A1'
+  solver1.spin = 0
+  solver2 = fci.direct_spin1_symm.FCI(mol)
+  solver2.wfnsym= 'A2'
+  solver2.spin = 2
+
+  mycas = mcscf.CASSCF(mf, 4, 4)
+  mcscf.state_average_mix_(mycas, [solver1, solver2], weights)
+  mycas.kernel()
+
+See :source:`examples/mcscf/41-state_average.py` for a complete example.
+
+
+Job Control
+-----------
+
+Optimization Settings
+"""""""""""""""""""""
+
+For CASSCF calculations, users may want to modify several of the convergence tolerances such as the energy tolerance (``conv_tol``), the orbital gradient tolerance (``conv_tol_grad``), and the maximum number of MCSCF iteration (``max_cycle_macro``).
+
+.. code-block:: python
+
+  mycas = mcscf.CASSCF(mf, 6, 6)
+  mycas.conv_tol = 1e-12
+  mycas.conv_tol_grad = 1e-6
+  mycas.max_cycle_macro = 25
+  mycas.kernel()
+
+
+Initial Guess
+"""""""""""""
+
+Initial guess orbitals for the CASSCF calculation may be passed to the ``kernel`` member function of an MCSCF object.
+
+.. code-block:: python
+
+  mycas = mcscf.CASSCF(myhf, 8, 8)
+  mycas.kernel(my_custom_mos)
+
+
+CI coefficients from a previous calculation can also be passed as an initial guess to expedite the calculation:
+
+.. code-block:: python
+  mycas = mcscf.CASSCF(myhf, 8, 8)
+  mycas.kernel(my_custom_mos, my_custom_ci)
+
+Examples:
+
+* :source:`examples/mcscf/14-project_init_guess.py`
+* :source:`examples/mcscf/31-cr2_scan/cr2-scan.py`
+* :source:`examples/mcscf/34-init_guess_localization.py`
+* :source:`examples/mcscf/43-avas.py`
+* :source:`examples/mcscf/43-dmet_cas.py`
+
+
+Restarting
+""""""""""
+
+.. warning::
+  When running large calculations, it's always recommended that you specify a checkpoint file for your calculation.
+
+.. code-block:: python
+
+  mycas.chkfile = "casscf.chk"
+
+Much like :mod:`scf`, if a job is interrupted, users can restart the MCSCF calculations using checkpoint files from a crashed calculation.
```

Comment:
Another use is to start from smaller active space orbitals or different spin/symmetry sector.

Reply:
@bogdanoff great addition!